news/_posts/core-weekly/2019-05-07-coreweekly-week-18-2019.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 18 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-05-07 18:00:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 29th of April to Sunday 5th of May 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
```

Comment:
Maybe we could explain why there is so much PR that fix tests ? Related to the validation process for 1.7.6 ?

Reply:
That could be a very good news for the next core weekly, or, maybe, a dedicated article.